src/test/java/org/breedinginsight/api/auth/AuthServiceLoginHandlerUnitTest.java

```diff
@@ -51,31 +49,35 @@
     @Property(name = "web.cookies.login-redirect")
     private String loginRedirectCookieName;
 
+    private String userName = "1111-2222-3333-4444";
+
     @AfterAll
     public void finish() {
         super.stopContainers();
     }
 
+    //TODO: Fix this test
```

Comment:
Is this and the other todos for a future development or were they accidentally left in? 

(Also a+ re: cats)